packages/flutter/test/material/text_field_test.dart

```diff
@@ -3455,8 +3456,11 @@ void main() {
 
     expect(tapCount, 0);
     await tester.tap(find.byType(TextField));
+    await tester.pump(const Duration(milliseconds: 300));
```

Comment:
A comment that explained the significance of 300ms would be helpful (I assume 300ms > double tap interval).

Reply:
Done